src/index.js

```diff
@@ -19,21 +19,23 @@ export default function (options = {}, callback = function noop () {}) {
   let testAttempt = 1
   let parsedOptions = parseOptions(options)
   let parser = getParser(parsedOptions.parser)
+  let plainLogger = new Logger(null)
```

Comment:
Ah, gotcha! I definitely understand that. How about adding a separate method/level to `logger` instead of having to make an entirely new instance? Here's an [example](https://gist.github.com/NickTomlin/608aada51df220b97a0bb2d1e88c2c23#file-logger-js-L18-L39); that way, one can just use `logger.log` or `logger.protractor` depending on whether or not output should be colorized.